Workflows/GIT/Clean changes.md

```diff
@@ -76,6 +76,29 @@ While on feature branch:
 ```bash
 git fetch origin master
 git rebase -i origin/master
+```
+
+This will start interactive rebase. You can edit it through terminal or your editor of choice.
```

Comment:
the terminal also has an editor, so I'd keep it simpler :)

```suggestion
This will start interactive rebase. You can edit it with your editor of choice.
```